show/muxcable.py

```diff
@@ -27,6 +27,18 @@
 VENDOR_MODEL_REGEX = re.compile(r"CAC\w{3}321P2P\w{2}MS")
 
 
+def get_interface_alias(port, db):
+
+    if port is not "all" and port is not None:
+        alias = port
+        iface_alias_converter = clicommon.InterfaceAliasConverter(db)
+        port = iface_alias_converter.alias_to_name(alias)
+        if port is None:
+            click.echo("cannot find port name for alias {}".format(alias))
+            sys.exit(STATUS_FAIL)
+
+    return port
```

Comment:
Rather than reimplementing this function, can we share this from a common location?

Reply:
fixed